### ISSUE TITLE
Packages are not deleted despite the success message being returned

### ISSUE THREAD

**Issue by grzegorzkrukowski** [actions/delete-package-versions#78]
I used this action to delete over 250 packages in 3 runs due to 99 limitations per run.
Each run correctly showed 99 removals + the last one 52 - as it removed it.

But all packages are still visible in a browser when I check it here:
`https://github.com/OWNER/REPO/pkgs/npm/PACKAGE_NAME/versions`

Re-running the action is not removing anything anymore.

**Comment by AJ-7885** [comment]
I count the same issue it shows it is remove but when I check the published version they all remind as before.

**Comment by csaba-kovacs** [comment]
Same here, shows 6 but nothing was deleted

**Comment by peter-at-work** [comment]
https://github.com/actions/delete-package-versions/issues/74 seems related to this and the real cause of the problem

**Comment by csaba-kovacs** [comment]
Bump!

**Comment by KennyKYLam** [comment]
Deleting shows success for me but the total # of packages is the same 😢  and showed no increase in deleted packages.

**Comment by nadesu** [comment]
Yes, as mentioned [here](https://github.com/actions/delete-package-versions/issues/74), this Action does not work for `npm` registry atm and it throws stale results. We are actively working on getting it work again for `npm` registry along with `container registry`.

**Comment by AJ-7885** [comment]
I am try the same with Github registry and has same issue, where this should work?

**Comment by NeelimaShrm** [comment]
Just checking, by chance is this resolved?

**Comment by joseenriquedevathand** [comment]
I have the same issue over here, there are any update of this problem?

**Comment by peter-at-work** [comment]
With #74 being fixed/closed starting with v4.0.0 or later of the action, this issue can also be closed.  Would the OP or others provide a workflow YAML and/or run logs to look at and see if it's the same issue?
